crates/rpc/rpc-server-types/src/constants.rs

```diff
@@ -48,8 +48,9 @@ pub const DEFAULT_ENGINE_API_IPC_ENDPOINT: &str = "/tmp/reth_engine_api.ipc";
 /// The default eth historical proof window.
 pub const DEFAULT_ETH_PROOF_WINDOW: u64 = 0;
 
-/// Maximum eth historical proof window. Equivalent to roughly one month of data.
-pub const MAX_ETH_PROOF_WINDOW: u64 = 216_000;
+/// Maximum eth historical proof window. Equivalent to roughly one and a half months of data on a 12
+/// second block time, and a week on a 2 second block time.
+pub const MAX_ETH_PROOF_WINDOW: u64 = 302_400;
```

Comment:
could we use math here? 
like seconds month / blocktime?

Reply:
we could, although what blocktime do we base it off?

Reply:
originally we based it off `30 * 24 * 60 *60 / 12`. now it's `7 * 24 * 60 *60 / 2`